projects/vdk-plugins/vdk-ingest-http/tests/test_ingest_over_http.py

```diff
@@ -226,3 +229,31 @@ def arg(a):
         cert="cert.pem",
         verify=True,
     )
+
+
+@mock.patch("json.dumps")
+def test_ingest_over_http_json_dumps_parameters_propagation(mock_jsondumps):
+    allow_nan = True
+
+    def configuration_side_effect(*args, **kwargs):
+        def arg(a):
+            return args[0] == a
+
+        if arg("INGEST_OVER_HTTP_ALLOW_NAN"):
+            return allow_nan
+        return None
+
+    job_context = MagicMock()
```

Comment:
`job_context` is already mocked in the global scope at the top of the file. Do you need to mock it again?

Reply:
yes, it seems to me, since I customise the configuration_side_effect in a test-specific way;
same stuff already merged with `test_ingest_over_http_result` for example

Reply:
Then, do we need job_context mocked in the global scope?

Reply:
Yes, it is still used by the majority of other tests, that do not introduce test-specific patches